experiments/AMIP/moist_mpi_earth/slab_ice/slab_init.jl

```diff
@@ -0,0 +1,78 @@
+# slab_ice
+
+# sea-ice parameters
+struct IceSlabParameters
+    h::FT # sea ice height
+    ρ::FT
+    c::FT
+    T_base::FT
+    z0m::FT
+    z0b::FT
+    T_freeze::FT # temperature at freezing point [K] 
+    k_ice::FT   # thermal conductivity of ice [W / m / K]
```

Comment:
Note: the conservation tests all assume k_ice and k_land are zero, so that there is no conductive heat flux at the bottom.